File: ChartView/3DBarChart/3DBarChartVB/MyDataPointComparer.vb
Imports _3DBarChartVB
Imports Telerik.Charting

Public Class MyDataPointComparer
    Implements IComparer(Of DataPointModel)

    Public Function Compare(ByVal x As DataPointModel, ByVal y As DataPointModel) As Integer Implements IComparer(Of DataPointModel).Compare
        Return If(x.Year > y.Year, 1, -1)
    End Function

End Class

File: ChartView/3DBarChart/3DBarChart/MyDataPointComparer.cs
using System;
using System.Collections.Generic;
using Telerik.Charting;

namespace _3DBarChart
{
    public class MyDataPointComparer : IComparer<DataPointModel>
    {
        public int Compare(DataPointModel x, DataPointModel y)
        {
            return x.Year > y.Year ? 1 : -1;
        }
    }
}
